features/bookmarks/bookmark_create.feature

```diff
@@ -448,3 +448,13 @@ Scenario: A bookmark with duplicate tags other than capitalization has only firs
       And I fill in "Relationships" with "Relationship 1, Relationship 2"
       And I press "Create"
     Then I should see "Fandom, relationship, and character tags must not add up to more than 5. You have entered 6 of these tags, so you must remove 1 of them."
+
+  Scenario: Bookmark can be added to collection by Archivist without error (Issue 6872)
```

Comment:
You can name the scenario "Archivists can add bookmarks to collections". You don't have to mention "without error" (we only need to be specific if we expect to see errors) or the issue (we only need to if the purpose of the test is not obvious just from the scenario name).